pkgs/development/tools/kafkactl/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+let
+  owner = "deviceinsight";
+in
+  buildGoPackage rec {
+    pname = "kafkactl";
+    version = "1.20.1";
```

Comment:
[1.23.1](https://github.com/deviceinsight/kafkactl/releases/tag/v1.23.1) is the latest release.